packages/updatePrescriptionStatus/tests/test-handler.test.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {handler} from "../src/updatePrescriptionStatus"
+import {DynamoDBClient} from "@aws-sdk/client-dynamodb"
+import {mockClient} from "aws-sdk-client-mock"
+import exampleInCollectionLocker from "../../specification/examples/request-in-collection-locker.json"
+import exampleMultipleItems from "../../specification/examples/request-multiple-items.json"
+import exampleMissingFields from "../../specification/examples/request-missing-fields.json"
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda"
+
+mockClient(DynamoDBClient)
+
+const generateMockEvent = (body: any): APIGatewayProxyEvent => ({
+  body: JSON.stringify(body),
+  headers: {"x-request-id": "test-request-id"},
+  multiValueHeaders: {},
+  httpMethod: "POST",
+  isBase64Encoded: false,
+  path: "/",
+  queryStringParameters: null,
+  multiValueQueryStringParameters: null,
+  stageVariables: null,
+  requestContext: {} as any,
+  resource: "",
+  pathParameters: null
+})
+
+describe("Unit test for app handler", () => {
+  it("should return 201 with response bundle for a single item", async () => {
+    const event: APIGatewayProxyEvent = generateMockEvent(exampleInCollectionLocker)
+    const response: APIGatewayProxyResult = await handler(event, {} as any)
+
+    expect(response.statusCode).toBe(201)
+    const responseBody = JSON.parse(response.body!)
+    expect(responseBody).toHaveProperty("resourceType", "Bundle")
+  })
+
```

Comment:
Needs a test asserting that for a single item a single put request is sent to dynamodb, ideal ay to do this would be to use a local testing dynamo and assert that the item has been created, see: https://jestjs.io/docs/dynamodb

Reply:
The test successfully updates a single item in DynamoDB has been added.